spec/lib/forms/client_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Forms::Client do
+  subject { described_class.new(query) }
+
+  let(:query) { nil }
```

Comment:
You don't need to define it (:query) here since its in each context, which is a better idea in any case